core/src/main/java/ch/cyberduck/core/Profile.java

```diff
@@ -525,7 +525,7 @@ public String getOAuthClientId() {
     @Override
     public String getOAuthClientSecret() {
         final String v = this.value(OAUTH_CLIENT_SECRET_KEY);
-        if(StringUtils.isBlank(v)) {
+        if(null == v) {
```

Comment:
Why check for `null`? All other fields are checked against `StringUtils.isBlank()`.

Reply:
Because this allow to define a default empty client secret that skip the prompt. See cbb5a225 and 43dc41b3.